pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -108,6 +117,12 @@ type PostgresClusterSpec struct {
 	// +kubebuilder:validation:Maximum=13
 	PostgresVersion int `json:"postgresVersion"`
 
+	// The PostGIS extension version installed in the PostgreSQL container.
+	// When set, this value indicates that a PostGIS enabled image will be
+	// used when created the cluster.
```

Comment:
⛏️ I think this and `postgresVersion` should say "the PostgreSQL image" rather than "the PostgreSQL container".

❓ The field can be used even after the cluster is created, but it only has meaning when `image` is omitted... Can we get away with leaving that explanation in `image`?

```suggestion
	// The PostGIS extension version installed in the PostgreSQL image.
```

Reply:
I'll make the updates from 'container' to 'image'.

For field question, I do agree rewording may be helpful, but perhaps adding a note about it only being relevant when using a related image?

Reply:
I was thinking this would be appropriate:
```
	// The PostGIS extension version installed in the PostgreSQL image.
	// When image is not set, indicates a PostGIS enabled image will be used.
	// +optional
```